### ISSUE TITLE
indent: `def`

### ISSUE THREAD

**Issue by jbodah** [elixir-lang/vim-elixir#298]
# Actual

```ex
    def handle_call({:release_lock, key}, _from, state) do
      case get_lock(state, key) do
        nil ->
          {:reply, {:error, :already_unlocked}, state}

        _ ->
          new_state = delete_lock(state, key)
          {:reply, :ok, new_state}
      end
    end

      def
```

# Expected

```ex
    def handle_call({:release_lock, key}, _from, state) do
      case get_lock(state, key) do
        nil ->
          {:reply, {:error, :already_unlocked}, state}

        _ ->
          new_state = delete_lock(state, key)
          {:reply, :ok, new_state}
      end
    end

    def
```

**Comment by jbodah** [comment]
Waiting on fix to be pushed

**Comment by jbodah** [comment]
Fixed in 222381d
